schema/deploy/shipping/views.sql

```diff
@@ -3220,7 +3220,7 @@ create materialized view shipping.__uw_encounters as (
 	left join shipping.fhir_questionnaire_responses_v1 q_testing_trigger on q_testing_trigger.encounter_id = encounter.encounter_id and q_testing_trigger.link_id = 'testing_trigger'
 	left join shipping.fhir_questionnaire_responses_v1 q_surge_selected_flag on q_surge_selected_flag.encounter_id = encounter.encounter_id and q_surge_selected_flag.link_id = 'surge_selected_flag'
 	left join shipping.fhir_questionnaire_responses_v1 q_prior_test_positive_date on q_prior_test_positive_date.encounter_id = encounter.encounter_id and q_prior_test_positive_date.link_id = 'prior_test_positive_date'
-	where encounter.details @> '{"_provenance": {"redcap": {"url": "https://redcap.iths.org/", "project_id":23854, "event_name": "encounter_arm_1"}}}'
+	where encounter.details @> '{"_provenance": {"redcap": {"url": "https://hct.redcap.rit.uw.edu/", "project_id":45, "event_name": "encounter_arm_1"}}}'
```

Comment:
For encounters, since those aren't getting copied over, I think we'd want this to include both old and new server projects. BUT, if it does include both, it would be great to test how this will interact with the uw_priority_queue_v1 view below, used for selecting HCT participants for testing invitations. I think this would be great to run with some test data in the new project to see how it would handle different situations.